grafana/README_DOCKER.md

```diff
@@ -0,0 +1,28 @@
+# Docker Container for IoTDB Grafana Adapter
+
+## Build
+
+* `mvn clean package`
+* in the module root execute `docker image build -t apache/iotdb-grafana:{version} .` where version is e.g. 0.9.3.
```

Comment:
Hi, now we only have apache/iotdb repo...
If we need apache/iotdb-grafana, we need to apply a new repo from INFRA.